### PR TITLE
Feature/expanded_repo_view -> dev

### PR DESCRIPTION
**Pull request by GGLozanov**
Small additions to the repo scene to flesh out the repo statistics display with the other properties defined in the Repository model. Also made some small method changes and abstractions into protocols. Requested review.

### CODE REVIEW

**Comment by alexvidenov** [comment]
Abstracting the renderOptionalLabelText method in a protocol extension makes sense since eventually, it can be used a lot.

**Comment by alexvidenov** [comment]
The displaying of stars in the second StackView looks good, but I think the description has no place in a cell, the text is either really small or glued to the title, depending on the length of the description.

File: GitHubExplorer/GitHubExplorer/ViewControllers/ReposViewController.swift [hunk 1]
```diff
@@ -66,12 +66,24 @@ extension ReposViewController: UITableViewDataSource{
...
     @IBOutlet weak var repoName: UILabel!
+    @IBOutlet weak var repoDescription: UILabel!
+    @IBOutlet weak var repoForks: UILabel!
+    @IBOutlet weak var repoStars: UILabel!
     
     func setRepo(repo: Repository){
         repoName.text = repo.repoName
+        
+        let intRenderPredicate: (Int?) -> Bool = { value in
+            (value ?? -1) > 0 // (potential FIXME: clean this hack solution to nil returning false)
```
Comment (MoridinBG) [inline GitHubExplorer/GitHubExplorer/ViewControllers/ReposViewController.swift]: Don't make one liners for the sake of one liners.
```guard let value = value else { return false }```

File: GitHubExplorer/GitHubExplorer/Utils/WithOptionalUILabel.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import UIKit
+
+// Extracted if used later for other optional labels
+protocol WithOptionalUILabel {
+    
+}
+
+extension WithOptionalUILabel {
+    func renderOptionalLabelText<T>(label: UILabel, textValue: T?, prefix: String?, renderPredicate: (T?) -> Bool = { value in
```
Comment (MoridinBG) [inline GitHubExplorer/GitHubExplorer/Utils/WithOptionalUILabel.swift]: This make very little sense as an empty protocol with a default function. Better extract to a utility class, ad an explicit dependency on the utility class and use as an instance function of the utility.

**Review by MoridinBG — APPROVED** [review]

**Pull request merged**